web/style.css

```diff
@@ -45,12 +45,12 @@ html {
     border-left: 4px solid #536115;
 }
 
-#text > h1 {
+.links {
     text-align: center;
 }
 
-#text > div {
-    margin-left: 50px;
+#text > h1 {
+    text-align: center;
 }
 
```

Comment:
Deleted 
`#text > div {
    margin-left: 50px;
}`
What's the point of this? If it is related to examples, this can be added by 
`#examples {
    margin-left: 50px;
}`

Reply:
Sorry, I'm very bad at `css` and `js`. Probably, you can merge it, if you are good with `css` and you think this changes are correct.